### PR TITLE
RIA-4712 Edit Hearings - Edit list: Appellant - Edit listing notifica…

### PR DESCRIPTION
**Pull request by burale**
### JIRA link (if applicable) ###
[RIA-4712](https://tools.hmcts.net/jira/browse/RIA-4712)

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/resources/scenarios/RIA-4712-aip-edit-listing-notification.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          "PA/12345/2019",
+          "LP/12345/2019",
+          "Talha Awan",
+          "{$iaAipFrontendUrl}",
+          "Taylor House",
+          "1 May 2019",
+          "3 May 2019",
+          "14:25",
+          "Location: North Shields"
+        ]
```
Comment (bradsorour) [inline src/functionalTest/resources/scenarios/RIA-4712-aip-edit-listing-notification.json:84]: @burale you have checked for customer services details again in your functional tests....

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -221,12 +221,15 @@ govnotify:
         legalRep:
           email: e79249e2-ecbf-4d2b-8458-006285613402
     caseEdited:
+      homeOffice:
+        email: b4cfdb17-6130-4f52-aec4-f3571e030790
```
Comment (bradsorour) [inline src/main/resources/application.yaml:225]: @burale there is a new template here but where is being used / called ?

Comment (burale) [reply src/main/resources/application.yaml:225]: This is not new template and it is used in "HomeOfficeEditListingPersonalisation" class.

**Review by bradsorour — APPROVED** [review]
LGTM

**Pull request merged**